test/shared_setup.py

```diff
@@ -15,10 +16,52 @@ def test_get_name_url(self):
             'download_url': 'https://github.com/hpcugent/vsc-install/archive/0.1.2.tar.gz',
         }
         for fn in ['PKG-INFO', 'git_config', 'git_config_1', 'git_config_2', 'git_config_3', 'git_config_4']:
-            self.assertEqual(get_name_url(os.path.join(REPO_TEST_DIR, 'setup', fn), version='0.1.2'), res,
+            self.assertEqual(get_name_url(os.path.join(shared_setup.REPO_TEST_DIR, 'setup', fn), version='0.1.2'), res,
                              msg='determined name and url from %s file' % fn)
 
         res.pop('download_url')
         fn = 'git_config'
-        self.assertEqual(get_name_url(os.path.join(REPO_TEST_DIR, 'setup', fn), version='0.1.2', license_name='LGPLv2+'), res,
+        self.assertEqual(get_name_url(os.path.join(shared_setup.REPO_TEST_DIR, 'setup', fn), version='0.1.2', license_name='LGPLv2+'), res,
                          msg='determined name and url from %s file with license' % fn)
+
+    def test_sanitize(self):
+        """Test sanitize function"""
+        os.environ['VSC_RPM_PYTHON']='1'
+        self.assertEqual(sanitize('anything >= 5'), 'python-anything >= 5',
+                         msg='all packages are prefixed with VSC_RPM_PYTHON set')
+
+        self.assertEqual(sanitize('vsc-xyz >= 10'), 'python-vsc-xyz >= 10',
+                         msg='vsc packages are prefixed with VSC_RPM_PYTHON set')
+
+        self.assertEqual(sanitize('python-ldap == 11'), 'python-ldap == 11',
```

Comment:
This is a good test to add in. Can we also add `python_*` such as [`python_webdav`](https://pypi.python.org/pypi/python_webdav/0.4]